actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -359,6 +359,9 @@ def each
 
     # Send the file stored at +path+ as the response body.
     def send_file(path)
+      ActiveSupport::Deprecation.warn(
+        "send_file is deprecated with no replacement and will be removed in 7.1."
```

Comment:
I think this leaves users a bit high and dry. Downloading files is a pretty common task.

Reply:
That was my thought too but I'm glad I opened the PR and got this feedback so quick, thank you!